src/JWT/Jwt.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Linq;
+
+namespace JWT
+{
+    /// <summary>
+    /// A class that represent a JWT
+    /// </summary>
+    internal class Jwt
```

Comment:
`class Jwt` inside `namespace JWT` is quite confusing. Let's have a better name for it. My idea was `JwtPayload` with property `Body`. Another would be `JwtParts` with property `Payload`

Reply:
Also I'd make it public since there will be public overloads accepting it.

Reply:
Something like JwtModel?
Payload isn't correct because the is the word in the RFC document for the Payload part. It's maybe a little bit confusing.

Reply:
If the RFC uses the term Payload then we shall stick to it. Model is good when it's something like MVC or MVVP and even there I try not to add obvious suffixes, e.g. Model, Dto, etc. Maybe then just `JwtParts`?

Reply:
okay I use now JwtParts. The Enum renamed to JwtPartsPointer